runtime/oti/j9protos.h

```diff
@@ -1133,6 +1133,52 @@ extern J9_CFUNC struct J9Method* jitResolveStaticMethodRef(J9VMThread *vmStruct,
 extern J9_CFUNC struct J9Method* jitResolveSpecialMethodRef(J9VMThread *vmStruct, J9ConstantPool *constantPool, UDATA cpOrSplitIndex, UDATA resolveFlags);
 extern J9_CFUNC struct J9Class * jitGetDeclaringClassOfROMField(J9VMThread *vmStruct, J9Class *clazz, J9ROMFieldShape *romField);
 
+typedef struct J9MethodFromSignatureWalkState {
+	const char *className;
+	U_32 classNameLength;
+	struct J9JNINameAndSignature nameAndSig;
+	struct J9VMThread *vmThread;
+	struct J9ClassLoaderWalkState classLoaderWalkState;
+} J9MethodFromSignatureWalkState;
+
+/**
+ * Cleans up the iterator after matching all methods found.
+ *
+ * @param state The inlialized iterator state
+ */
+extern J9_CFUNC void
+allMethodFromSignatureEndDo(J9MethodFromSignatureWalkState *state);
+
+/**
+ * Advances the iterator looking for the next method matching the full signature provided during initialization.
+ *
+ * @param state The inlialized iterator state
+ *
+ * @return      J9Method matching the full signature provided during initialization of the iterator in the class loader
+ *              if it exists; NULL if no such method is found.
+ */
+extern J9_CFUNC struct J9Method *
+allMethodFromSignatureNextDo(J9MethodFromSignatureWalkState *state);
+
+/**
+ * Initializes the iterator state and returns the first method matching the supplied signature from any class loader.
```

Comment:
This should note that wildcards are not supported in `className`, `methodName` or `methodSig`.

Reply:
Fixed in a36864f